src/js/player.js

```diff
@@ -1292,7 +1292,15 @@ class Player extends Component {
    * @method play
    */
   play() {
-    this.techCall_('play');
+    // Only calls the tech's play if we already have a src loaded
+    if (this.player_.currentSrc()) {
```

Comment:
You should be able to just do `this.currentSrc()` here, since the context is already the player itself. We also may want to do `this.src() || this.currentSrc()` to make sure that we catch issues where the `src` is set but `currentSrc` isn't (can happen when adblock is involved.